### PR TITLE
not support postbuild incremental now

### PR DESCRIPTION
**Pull request by ansyral**
@vwxyzh @superyyrrzz @hellosnow @qinezh @chenkennt

### CODE REVIEW

**Comment by vwxyzh** [comment]
:+1:

File: src/Microsoft.DocAsCode.Build.Engine/SingleDocumentBuilder.cs [hunk 1]
```diff
@@ -373,7 +373,7 @@ into model
                 MarkdownService = markdownService,
                 DependencyGraph = new DependencyGraph(),
             };
-            BuildCore(new List<HostService> { hostService }, parameters.MaxParallelism, null, null, null, null);
+            BuildCore(new List<HostService> { hostService }, parameters.MaxParallelism, null, null);
```
Comment (herohua) [inline src/Microsoft.DocAsCode.Build.Engine/SingleDocumentBuilder.cs]: maybe use a switch to turn it off rather than delete the code?

**Comment by hellosnow** [comment]
:shipit:

**Comment by herohua** [comment]
👍

File: src/Microsoft.DocAsCode.Build.Engine/SingleDocumentBuilder.cs (hunk 1 above)
Comment (ansyral) [inline src/Microsoft.DocAsCode.Build.Engine/SingleDocumentBuilder.cs]: i didn't delete the updater method. it is easy to change the code later

**Pull request merged**